### PR TITLE
Add custom.css for docs to change pre tag styles

### LINKED ISSUE #4238: Assembly functions on that docs page are truncated
There's a style called .wy-nav-content that is causing the Assembly explanations table to truncate at 800px, so you can't read the function blurbs unless you go into the browser debugger and turn off the max-width: 800px style attribute:

http://solidity.readthedocs.io/en/v0.4.21/assembly.html

However, even with that attribute turned off, the really long explanations still truncate.  It would be better to adjust the styles on that page so the function explanation text wraps and doesn't get hidden if it overflows.

### PR DESCRIPTION
**Pull request by shad-k**
Added word wrap styles for pre tag to display function details in docs without horizontal scroll or overflow to the right.

Fixes #4238

### CODE REVIEW

**Comment by chriseth** [comment]
I'm sorry, I just merged https://github.com/ethereum/solidity/pull/4368 before I saw your solution. Could you explain the difference in the approaches?

**Comment by shad-k** [comment]
Hi @chriseth
I have also stopped the code chunks from flowing to the right. Other than that there is not much change in the css.

There is also a difference in how the css file is imported. This is what I used as a reference to go about this:
http://docs.readthedocs.io/en/latest/guides/adding-custom-css.html

**Comment by axic** [comment]
I'm not proficient in sphinx, but the `sphinx.add_stylesheet` option seems cleaner than the merged solution

**Comment by shad-k** [comment]
Hi @chriseth
How are you planning to close this?

File: docs/conf.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def setup(sphinx):
     thisdir = os.path.dirname(os.path.realpath(__file__))
     sys.path.insert(0, thisdir + '/utils')
     from pygments_lexer_solidity import SolidityLexer
+    sphinx.add_stylesheet('css/custom.css')
```
Comment (chriseth) [inline docs/conf.py]: Can you move that to the bottom of the function? Everything else concerns setting up the lexer. Perhaps even add a blank line in between.

**Comment by chriseth** [comment]
@shad-k could you rebase and undo the other change?

**Comment by shad-k** [comment]
@chriseth done.

**Comment by shad-k** [comment]
@chriseth does it look okay to you now? I incorporated the changes you mentioned as well.

**Comment by chriseth** [comment]
Wonderful! Could you squash it into a single commit, please?

Ah, and there seems to be a build error, I think you did not undo those changes here: https://github.com/ethereum/solidity/pull/4368/files#diff-85987f48f1258d9ee486e3191495582dR149

**Comment by shad-k** [comment]
@chriseth done. Have undone the code and squashed the commits.

**Comment by chriseth** [comment]
Thanks!

**Pull request merged**